server/graphql/types/src/types/item_direction.rs

```diff
@@ -0,0 +1,142 @@
+use async_graphql::*;
+use graphql_core::simple_generic_errors::InternalError;
+use repository::{item_direction::ItemDirection, ItemDirectionRow};
+use service::ListResult;
+
+#[derive(PartialEq, Debug)]
+pub struct ItemDirectionNode {
+    item_direction: ItemDirection,
+}
+
+#[derive(SimpleObject)]
+pub struct ItemDirectionConnector {
+    total_count: u32,
+    nodes: Vec<ItemDirectionNode>,
+}
+
+#[Object]
+impl ItemDirectionNode {
+    pub async fn id(&self) -> &str {
+        &self.item_direction.item_direction_row.id
+    }
+
+    pub async fn item_link_id(&self) -> &str {
```

Comment:
I think we've agreed that in the API we should only expose `item_id` not `item_link_id`.
I've been asking for these things to be documented but yeah...

Reply:
https://github.com/msupply-foundation/open-msupply/pull/6408/commits/43151d5a6f4b78d7c3c9de63b079d5ed1616b589